src/otg/clump.py

```diff
@@ -61,8 +61,18 @@ def __post_init__(self: ClumpStep) -> None:
                 study_index=study_index, ld_index=ld_index
             ).clump()
         else:
+            if self.study_index_path is not None:
+                # Generate a list of study identifiers that we want to ingest:
+                study_ids_to_ingest = StudyIndex.from_parquet(
```

Comment:
My understanding is that the final study index will contain the blacklisted studies, with a flag (`isEligible`). Is that correct?
In order to make the `clump` step lighter, I'd use here the file you output with the list of blacklisted IDs. You read everything, and you filter out the ones in the file.What do you think?

Reply:
No. @d0choa and @addramir proposed to completely drop the non-eligible studies altogether.

Reply:
Also, I will split that step into two, too many things are happening inside and very confusing.